backend/library/utils.py

```
@@ -135,17 +129,17 @@ def load(
                 urn=self.data["target_requirement_urn"].lower(), default_locale=True
             )
         except RequirementNode.DoesNotExist:
-            err_msg = f"ERROR: target requirement with URN {self.data['target_requirement_urn']} does not exist"
-            print(err_msg)
-            raise Http404(err_msg)
+            error_msg = f"ERROR: target requirement with URN {self.data['target_requirement_urn']} does not exist"
+            print(error_msg)
+            raise Http404(error_msg)
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)